core/src/main/java/hudson/model/AbstractProject.java

```diff
@@ -1384,6 +1384,17 @@ public PollingResult poll( TaskListener listener ) {
      * {@link #poll(TaskListener)} method without the try/catch block that does listener notification and .
      */
     private PollingResult _poll(TaskListener listener, SCM scm) throws IOException, InterruptedException {
+
+        LOGGER.fine("Polling SCM changes of " + getName());
```

Comment:
This code existed before, I just moved it, and didn't changed to make the changes more obvious. 
If you want to review all jenkins code to mark all lines as :bug: then we will end up with a billion :bug:s. If you wish to make code cleanup PR, feel free to.